src/styles/blocks/block.scss

```diff
@@ -0,0 +1,31 @@
+.block {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax($height, 1fr));
+  grid-template-rows: repeat(auto-fill, minmax($height, 1fr));
```

Comment:
`grid-template-rows` isn't needed here, cause you've already set block height in `&__item`